test/e2e/framework/kubelet_stats.go

```diff
@@ -452,29 +453,7 @@ func TargetContainers() []string {
 	}
 }
 
-// ContainerResourceUsage is a structure for gathering container resource usage.
-type ContainerResourceUsage struct {
```

Comment:
What's the reason for only moving `ContainerResourceUsage`? What about the rest of this file?

Reply:
Moving the whole file will cause too many modifications. I think this patch should mainly focused on perf_util.go. For the rest of this file, I will give another patch after this patch merged, WDYT?

Reply:
sgtm